client/inbox-panel/card.js

```diff
@@ -116,9 +117,13 @@ class InboxNoteCard extends Component {
 			'components-popover__content',
 		];
 		// This line is for IE compatibility.
-		const relatedTarget = event.relatedTarget
-			? event.relatedTarget
-			: document.activeElement;
+		let relatedTarget;
+		if ( event.relatedTarget ) {
+			relatedTarget = event.relatedTarget;
+		} else if ( this.toggleButtonRef.current ) {
+			const ownerDoc = this.toggleButtonRef.current.ownerDocument;
+			relatedTarget = ownerDoc ? ownerDoc.activeElement : null;
```

Comment:
This fixes a lint error that doesn't like the use of `document.activeElement` and suggests to use `ownerDocument` instead.